app/views/components/_published-dates.html.erb

```diff
@@ -7,21 +7,20 @@
   history_class = "app-c-published-dates--history" if history.any?
 %>
 <% if published || last_updated %>
-<div class="app-c-published-dates <%= history_class %>" <% if history.any? %>id="history" data-module="toggle"<% end %>>
+<div class="app-c-published-dates <%= history_class %>" <% if history.any? %>id="history" data-module="toggle"<% end %> lang="en">
   <% if published %>
-    <%= t('components.published_dates.published', date: published) %>
+    Published <%= published %>
   <% end %>
   <% if last_updated %>
-    <% if published %><br /><% end %><%= t('components.published_dates.last_updated', date: last_updated) %>
+    <% if published %><br /><% end %>Last updated <%= last_updated %>
     <% if link_to_history && history.empty? %>
-      &mdash; <a href="#history"><%= t('components.published_dates.see_all_updates') %></a>
+      &mdash; <a href="#history">see all updates</a>
```

Comment:
Rather than hardcode the text here, we can be explicit about which locale translation we want to use: `<%= t('components.published_dates.see_all_updates', locale: :en) %>`